skyd/api.go

```diff
@@ -68,12 +71,12 @@ func NewAPI(nginxHost string, nginxPort int, skydHost, skydPassword string, skyd
 }
 
 // BlockSkylinks will perform an API call to skyd to block the given skylinks
-func (api *api) BlockSkylinks(sls []string) error {
+func (api *api) BlockHashes(hashes []string) error {
```

Comment:
Hm ok, I figured this was a good cut-off point because the skyd API expects strings. The blocker actually uses a slice of hashes everywhere, so right before calling this method I convert them because I figured it made sense to have this be the cut-off point.

I don't feel that strongly though so I don't mind updating! 👍

Reply:
I ended up keeping this because the `skyapi.SkynetBlocklistPOST{` struct expects a `[]string` and also because we would otherwise be adding in "our" types in the skyd API. I don't think it makes sense any more now that we've wrapped the `crypto.Hash` into our own type.

Since this was a `nit` I figured you'd be fine with that.

Reply:
fwiw the correctness of the hashes is asserted by the marshaling